src/components/Home/MovieCard.tsx

```diff
@@ -4,50 +4,60 @@ import styled from 'styled-components';
 
 import { IcHeartOn } from '../../assets/icon';
 import img_all from '../../assets/image/img_all.png';
+import { patchLikeData } from '../../libs/like';
+import { movieInfoTypes } from '../../types/movieInfo';
 
-export interface MovieCardProps {
-  id: number;
-  posterImg: string;
-  title: string;
-  ranking: string;
-  audience: string;
-  like: number;
-}
+const MovieCard = ({ movie_id, title, poster_url, ranking, total_audience, like_count }: movieInfoTypes) => {
 
-const MovieCard = ({ posterImg, title, ranking, audience, like }: MovieCardProps) => {
   const [selectedCard, setSelectedCard] = useState<boolean>(false);
   const [isLike, setIsLike] = useState<boolean>(false);
+  const [likeCount, setLikeCount] = useState<number>(like_count);
 
   const handleCard = () => {
     setSelectedCard(!selectedCard);
   };
 
   const handleButton = (event: React.MouseEvent<HTMLButtonElement>) => {
     event.stopPropagation();
-    setIsLike(!isLike);
+    const patchData = async () => {
+      const { message, isError } = await patchLikeData(movie_id);
+      if (!isError) setIsLike(!isLike);
+      if (message === '영화 좋아요에 성공했습니다.') {
+        setLikeCount(likeCount + 1);
+      } else {
+        setLikeCount(likeCount - 1);
+      }
+    };
```

Comment:
```suggestion
     message === '영화 좋아요에 성공했습니다.' ? setLikeCount(likeCount +1) : setLikeCount(likeCount - 1)
```


p2) 이렇게 삼항연산자 사용해줘도 좋을것 같아요!

Reply:
웅웅 그게 훨씬 보기 좋다! 고칠게유 감사함니당🔥